src/explore-education-statistics-admin/src/pages/edit-release/ReleasePublishStatusPage.tsx

```diff
@@ -13,28 +11,9 @@ const ReleasePublishStatusPage = ({
 }: RouteComponentProps<MatchProps>) => {
   const { releaseId } = match.params;
 
-  const [release, setRelease] = useState<Release>();
-
-  const [publicationTitle, setPublicationTitle] = useState('');
-
-  useEffect(() => {
-    const selectedRelease = DummyPublicationsData.getReleaseById(releaseId);
-
-    const owningPublication = DummyPublicationsData.getOwningPublicationForRelease(
-      selectedRelease,
-    );
-
-    setRelease(selectedRelease);
-
-    setPublicationTitle(owningPublication ? owningPublication.title : '');
-  }, [releaseId]);
-
   return (
-    <ReleasePageTemplate
-      publicationTitle={publicationTitle}
-      releaseId={releaseId}
-    >
-      {release && <h2 className="govuk-heading-m">Set publish status</h2>}
+    <ReleasePageTemplate publicationTitle="TODO" releaseId={releaseId}>
```

Comment:
I guess we don't have the publicationTitle

Reply:
Not done the service calls for this component yet, but when I do I will wire this up as per the other sections worked on so far